src/ImportDefinitionsBundle/Interpreter/AssetUrlInterpreter.php

```diff
@@ -52,14 +53,15 @@ public function interpret(Concrete $object, $value, Mapping $map, $data, Definit
         $path = $configuration['path'];
 
         if (filter_var($value, FILTER_VALIDATE_URL)) {
-            $filename = File::getValidFilename(basename($value));
+            $assetsUrlPrefix = PimcoreTool::getHostUrl() . str_replace(PIMCORE_WEB_ROOT, '', PIMCORE_ASSET_DIRECTORY);
+            if (0 === strpos($value, $assetsUrlPrefix)) {
+                // URL seems to be pointing to our own asset URL, try to lookup by that
+                $assetFullPath = str_replace($assetsUrlPrefix, '', $value);
 
-            // Convert placeholder path
-            $context = new PlaceholderContext($data, $object);
-            $assetPath = $this->placeholderService->replace($path, $context);
-            $assetFullPath = sprintf('%s/%s', $assetPath, $filename);
-            
-            if ($configuration['deduplicate_by_url']) {
+                $asset = Asset::getByPath($assetFullPath);
```

Comment:
This can be `null` so need to account for that.

Reply:
we should, otherwise it will throw an null-exception ;)

Reply:
Indeed. :)